### DISCUSSION TITLE
How to I secure my prompt flow? Is there a API-Key feature

### DISCUSSION THREAD

**Discussion by SteveSitekitcare** [microsoft/promptflow#3865] [Q&A]
Hi 

I have flow building and deploying in devops, but it isnt secure like the flows deployed in azure ai (ai foundry) where there is a api-key.

Is there a method of deploying the prompt flows with an APIKey?

Regards

**Comment by mrn-linak** [comment]
You can use Azure API Management and subscriptions to achieve something like this.
